ocs_ci/utility/rosa.py

```diff
@@ -596,3 +599,59 @@ def is_odf_addon_installed(cluster_name=None):
         return True
     else:
         return False
+
+
+def post_onboarding_verification():
+    """
+    Check that after onboarding consumer the relevant resources
+    were created in the provider cluster and then switch back to the original cluster:
+    1. StorageConsumer with the correct id exists and all its CephResources
+    are in Ready status
+    2. CephBlockPool and Subvolumegroup with the correct id are in Ready status
+    """
+    if config.multicluster:
+        restore_ctx_index = config.cur_index
+
+        consumer_ids = []
+        for cluster in config.clusters:
```

Comment:
To get all the consumer indexes, you can use the function `get_consumer_indexes_list`: https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/framework/__init__.py#L231.

Reply:
Then you can iterate over the consumer indexes list and switch to the consumer. The change is not critical.

Reply:
Thanks, done